src/web/components/Lazy.tsx

```diff
@@ -33,13 +26,12 @@ export const Lazy = ({ children, margin }: Props) => {
         return null;
     }
 
-    // Why do we check to see if we're in an iframe here?
+    // Why do we check to see if we're irunning in Chromatic here?
```

Comment:
```suggestion
    // Why do we check to see if we're running in Chromatic here?
```

Reply:
I _just_ changed that - we're really in sync today!

Reply:
Remote working for you, got to use the extra telepathic power